### DISCUSSION TITLE
Propozycja nowej listy - filtry blokujące pornografię

### DISCUSSION THREAD

**Discussion by PolishRoboDog** [FiltersHeroes/KAD#1861] [Pomysły]
### Dodatkowe informacje mogące mieć znaczenie
Czy PolishFIltersTeam byłby zainteresowany w ogóle stworzyć taką odrębną listę? Wydaje mi się, że jest dość duża luka jeśli o takie listy chodzi, zwłaszcza zawierajace polskojęzyczne strony. 

Zgłoszenie opublikowane anonimowo przez użytkownika.

**Comment by krystian3w** [comment]
Ja raczej nie, zostaje więc blocker / hawkeye / azet12.

**Reply by ClusterByte-x** [reply]
Jeśli byłoby zaplecze techniczne (repo, itd.) to mógłbym pomóc w wyszukiwaniu i uzupełnianiu listy. Ale czy to nie jest trochę walka z wiatrakami?

**Reply by krystian3w** [reply]
W sumie każda lista do blokowania kontentu to walka z wiatrakami, jak nie uruchamiamy każdego filtru jako generyk.

**Reply by hawkeye116477** [reply]
Ja też nie. A azet raz na parę lat się odzywa, więc pewnie też nie.

**Comment by blocker999** [comment]
Ja na razie nie planuję żadnej nowej listy, jak wyżej słusznie zauważono to walka z wiatrakami, bo tu język raczej nie ma znaczenia, ale jak ktoś chce to może taką listę stworzyć.
